pkg/daemon/plugin.go

```diff
@@ -23,10 +23,11 @@ type VendorPlugin interface {
 
 var pluginMap = map[string]string{
 	"8086": "intel_plugin",
-	"15b3": "mellanox_plugin",
+	"15b3": mellanoxPlugin,
```

Comment:
Why do we need this change?

Reply:
It's better to has it as a constant instead of hardcoded , since it is used in more than one place